tests/test_message_processing.py

```
@@ -2,96 +2,82 @@
 
-    # Start the bot
-    await client.start(discord_token)
-    await client.wait_until_ready()
-
-    # Verify that messages were processed
-    assert len(messages_processed) > 0, "No messages were processed"
-    assert messages_processed[0]["content"], "Message content is empty"
+    try:
+        await client.start(config.discord_token)
+        await asyncio.wait_for(test_completed.wait(), timeout=30.0)
+    except asyncio.TimeoutError:
```

Ruff (0.12.2)

Replace aliased errors with `TimeoutError`

Replace `asyncio.TimeoutError` with builtin `TimeoutError`

(UP041)